kolibri/core/settings.py

```diff
@@ -0,0 +1,27 @@
+"""
+Kolibri core settings module
+============================
+
+This module is *not* the project settings but defines variables that can be
+configured through the project settings - and their defaults.
+
+If something in the core uses a custom project setting, it should use a
+naming scheme in django.conf.settings like ``KOLIBRI_*``.
+
+Please avoid depending on settings in kolibri.deployments.default.settings,
+rather define the default here so we don't have multiple layers of defaults.
+"""
+
+from django.conf import settings
+
+# : These plugins are automatically enabled if they exist at the first run of
+# : Kolibri. After that, they are not searched for automatically since that would
+# : require explicitly disabling them in case they were to be removed on purpose
+# : from INSTALLED_APPS.
+WELL_KNOWN_PLUGINS = getattr(
+    settings,
+    'KOLIBRI_WELL_KNOWN_PLUGINS',
+    [
+        'kolibri.plugins.example_plugin',
+    ]
+)
```

Comment:
I'm a bit unlear what the intent of a setting like `KOLIBRI_WELL_KNOWN_PLUGINS` is. Are these things that are enabled by default? What happens when they're disabled? The naming is a bit ambiguous to me.

One piece of anecdotal experience: on a couple occasions now I've run into issues during dev (after switching branches) because the "installed plugins" information is stored outside of source control, in _~/.kolibri_.

Perhaps Kolibri could come with a `default_kolibri_settings.json` file checked in the root which specifies which plugins are installed, and then the `kolibri plugin enable`/`disable` commands might override these settings and operate on the _~/.kolibri_ file?